typings/balena-sdk/index.d.ts

```diff
@@ -698,8 +707,12 @@ declare namespace BalenaSdk {
 				ping(uuidOrId: string | number): Bluebird<void>;
 				getStatus(uuidOrId: string | number): Bluebird<string>;
 				getProgress(uuidOrId: string | number): Bluebird<number | null>;
-				lastOnline(device: Device): string;
-				getOsVersion(device: Device): string;
+				lastOnline(
+					device: AtLeast<Device, 'last_connectivity_event' | 'is_online'>,
+				): string;
+				getOsVersion(
+					device: AtLeast<Device, 'os_variant' | 'os_version'>,
+				): string;
```

Comment:
Is there any benefit over just doing `Pick`?

Reply:
Yes, if you do `Pick` then passing a full `Device` results in excess property errors